versions.yaml

```diff
@@ -153,14 +153,12 @@ assets:
   kernel:
     description: "Linux kernel optimised for virtual machines"
     url: "https://cdn.kernel.org/pub/linux/kernel/v5.x/"
-    uscan-url: >-
-      https://mirrors.edge.kernel.org/pub/linux/kernel/v5.x/linux-(5\.4\..+)\.tar\.gz
     version: "v5.10.25"
 
   kernel-experimental:
     description: "Linux kernel with virtio-fs support"
     url: "https://cdn.kernel.org/pub/linux/kernel/v5.x/"
-    tag: "v5.10.25"
+    tag: "v5.13.10"
```

Comment:
I believe you need `patches/v5.13.x/` with this change? At least for me `setup` did not finish properly (it did not generate `.config`)  until I added `patches/v5.13.x/no_patches`